website/server/controllers/api-v3/groups.js

```diff
@@ -127,7 +127,7 @@ api.getGroups = {
     if (validationErrors) throw validationErrors;
 
     let types = req.query.type.split(',');
-    let groupFields = basicGroupFields.concat(' description memberCount balance');
+    let groupFields = basicGroupFields.concat(' description memberCount balance purchased');
```

Comment:
Yes, group.purchased has some private data that should only be shown to the paying user. You can use a virtual property to show the subscription status